konch.py

```diff
@@ -287,7 +287,7 @@ def get_obj_name(obj: typing.Any) -> str:
     try:
         name = obj.__name__
     except AttributeError:
-        name = ""
+        name = obj.__hash__
```

Comment:
I don't think using `__hash__` is correct. Since it won't be a valid Python identifier, there'd be no way to actually use the object in the shell, right?

Reply:
Great point, you are right again.  :)